docs/user-guide/gaffer-basics/what-is-gaffer.md

```diff
@@ -21,4 +22,24 @@ flowchart TD
         B --> C(Gaffer)
         C --> D(key-value store)
     end
-```
\ No newline at end of file
+```
+
+!!! note
+    Knowledge of the [Gaffer schema](../schema.md) in use is usually required to
+    formulate a graph query but technically optional hence the dotted connection.
+
+## Why would you want to use Gaffer?
+
+Conceptually a graph database/data platform can have a large variety of uses
+where it'll provide specific advantages over other data storage/analytic
+platforms which will be discussed in the ["What is Graph?"](./what-is-a-graph.md)
+section of these docs. Gaffer provides an extensible and straightforward way to
+insert, manage and query the graph data stored in the underlying storage systems
+(ie. Accumulo) that is native to the community.
```

Comment:
"native to the community" - I don't understand this

Reply:
I think it doesn't really add much to the sentence so will remove